app/code/Magento/Catalog/view/base/web/js/price-utils.js

```diff
@@ -79,7 +79,7 @@ define([
         am = Number(Math.round(Math.abs(amount - i) + 'e+' + precision) + ('e-' + precision));
         r = (j ? i.substr(0, j) + groupSymbol : '') +
             i.substr(j).replace(re, '$1' + groupSymbol) +
-            (precision ? decimalSymbol + am.toFixed(2).replace(/-/, 0).slice(2) : '');
+            (precision ? decimalSymbol + am.toFixed(precision).replace(/-/, 0).slice(2) : '');
```

Comment:
Shouldn't we slice `precision` digits as well?

Reply:
@orlangur 
the slice is applied not on the precision but on the price before digits for ex.

0.4880
the slice will remove the price and it will retun the decimals like this :  
4880

in this case it should be always 2

Reply:
Ok, thanks for explanation as I didn't try to understand this logic in deep :)